docs/reference/buildx_build.md

```diff
@@ -202,15 +206,17 @@ More info about cache exporters and available attributes: https://github.com/mob
 ```
 
 Export build cache to an external cache destination. Supported types are
-`registry`, `local`, `inline` and `gha`.
+`registry`, `local`, `inline`, `gha` and `s3`.
```

Comment:
:tada: Good to have some more docs! I think we probably want to update the `README.md` as well alongside this?

Reply:
Ah yes looks like it's missing